syfertext/token.py

```diff
@@ -7,7 +8,7 @@
 
 
 class Token:
-    def __init__(self, doc, token_meta: "TokenMeta"):
+    def __init__(self, doc, token_meta: "TokenMeta", i: int):
```

Comment:
Please put `i` inside `token_meta`. Also, call it something more expressive, such as `position`

Reply:
Do you mean place i before token_meta, otherwise putting i inside TokenMeta class would be a very complex step considering tokenizer, and might even break it.

Reply:
It is indeed difficult to add positional information to the `TokenMeta` class. With its use of other methods that don't have access to the `doc` object while creating `TokenMeta`s, examples:

`_get_prefix_token_meta`
`_get_infix_token_metas`
`_get_exception_token_metas`

I added the position information to `token_meta` instead. 

Did you mean we should change `TokenMeta` to be able to track its position in a document?

Reply:
Oh @Nilanshrajput @Akramz I agree with you. No need to put it inside TokenMeta. My bad! This is just the Token. You can put it back in the constructor as an argument called 'position'. Sorry for that